WtWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -1133,33 +1131,35 @@
 }
 
 36073 = {
-	name = "Bonifacius"	#PLRE vol. 2 Bonifacius 3
-	# Gaius Bonifacius
+	name = "Bonifatius"	#PLRE vol. 2 Bonifacius 3
 	dynasty = 10424629
 	religion = "nicene"
 	culture = "roman"
 	384.1.1 = {
 		birth = yes
 	}
+	422.1.1 = {
+		add_spouse = 36074
+	}
 	432.1.1 = {
 		death = yes
 	}
 }
 
 36074 = {
-	name = "Pelagia"	#PLRE vol. 2 Pelagia 1
-	# Pelagia Naevia
-	dynasty = 10424630
-	religion = "nicene"
-	culture = "roman"
+	name = "Pelagia" # PLRE vol. 2 Pelagia 1
 	female = yes
-	384.1.1 = {
+	dynasty = 1042165
+	religion = "gothic_church"
+	culture = "visigothic"
+	father = 190115 # Berimund
```

Comment:
What is this based on? The PLRE doesn't mention Berimund as the father, just possibly of royal family

Reply:
I attach here a historiographical work that explains all this matter. Basically, Aetius married a noble visigothic woman who was not the daughter or kin of king Theodoric Balthing. The most pertinent thing for me was to assign it to the visigtohic branch of the amales of which Berimund is the main representative of it around the year 439 (date of the marriage between Aetius and Pelagia). Main primary sources are a poem by Merobaudes, Marcellinus Comes and Sidonius Apollinaris.

Reply:
https://archive.org/details/sim_transactions-of-the-american-philosophical-society_1971-01_61_1/page/30/mode/2up